music/scripts/generate-docs.sh

```diff
@@ -49,3 +59,16 @@ for path in ./*.html; do
     sed -i "" "s%${search}%${replace}%g" $path
   fi
 done
+
+cd $currDir
+yarn build-demos
+mkdir -p $TMP_DIR/demos && cp demos/*.{js,html,mid,css} $TMP_DIR/demos
+git checkout gh-pages
+docsDir=$(git rev-parse --show-toplevel)
+cd $baseDir
+git rm -fr music
+rsync -a $TMP_DIR/ music/
+git add music
+git commit -m "Updating music docs: $currDate"
```

Comment:
i would create that currDate variable here, not up top (i forgot what it did already)